adoptions.md

```diff
@@ -10,11 +10,18 @@ By clicking on the images below, you will be linked either to articles that
 cover the TUF adoption, presentations on the adoption, or to repositories
 containing the relevant code.
 
+<a
+href="https://github.com/awslabs/tough"
+rel="AWS Labs logo"><img src="/assets/images/amazon_logo.png"
+width="200"></a>
+<a href="https://fuchsia.dev/"
+rel="Google Fuchsia logo"><img src="/assets/images/Google_fuchsia3_logo.png"
```

Comment:
Let's use the Google logo instead.  It's more recognizable.

Reply:
Ok. I can do that.

Reply:
OK. Logo substitution done.